migrator/migrations/m_209_to_m_210_add_updated_at_to_network_flows_v2/stores/previous/store.go

```diff
@@ -0,0 +1,630 @@
+package previous
```

Comment:
If these are only for test, I personally prefer they go in a `test` directory.  We are not consistent with that as in the past some didn't agree.

Reply:
Done